FlowCrypt/src/main/java/com/flowcrypt/email/ui/activity/fragment/CreateMessageFragment.kt

```diff
@@ -1856,24 +1934,49 @@ class CreateMessageFragment : BaseFragment<FragmentCreateMessageBinding>(),
   }
 
   private fun fixNoKeyAvailableIssue(text: String) {
-    showActionDialogFragment(
-      navController,
-      requestKey = REQUEST_KEY_FIX_NO_PRIVATE_KEY_AVAILABLE,
-      dialogTitle = text,
-      isCancelable = true,
-      items = listOf(
-        DialogItem(
-          iconResourceId = R.drawable.ic_import_user_public_key,
-          title = getString(R.string.import_private_key),
-          id = RESULT_CODE_IMPORT_PRIVATE_KEY
-        ),
-        DialogItem(
-          iconResourceId = R.drawable.ic_edit_key_add_user_id,
-          title = getString(R.string.add_email_to_existing_key),
-          id = RESULT_CODE_ADD_USER_ID_TO_EXISTING_PRIVATE_KEY
+    if (account?.clientConfiguration?.usesKeyManager() == true) {
```

Comment:
@sosnovsky Please pay attention here. I've disabled the ability to use new changes for users where `account?.clientConfiguration?.usesKeyManager() == true`. I propose to discuss it a bit later and make changes via a separate PR.

Reply:
I think for now we can leave your implementation - as it's simpler, and if there will be any feedback asking for ability to fix keys even when using key manager - we'll work on it.